articles/Setup-Windows-Python.md

```diff
@@ -38,7 +38,12 @@ The default CNTK math library is the [Intel Math Kernel Library (Intel MKL)](htt
 
 ### OPTIONAL. OpenCV
 
-CNTK 2.2 requires [Open Source Computer Vision (OpenCV)](http://opencv.org/) to be installed but it is optional for CNTK 2.3+. Follow [this page](./Setup-OpenCV-on-Windows.md) to install it.
+CNTK 2.2 requires [Open Source Computer Vision (OpenCV)](http://opencv.org/) to be installed, but it is optional for CNTK 2.3+. Follow [this page](./Setup-OpenCV-on-Windows.md) to install it.
+
+You will need to install OpenCV for CNTK 2.3+ if you want to use the following components:
```

Comment:
Great info, I was wondering what are the components that require OpenCV :)
Do you happen to know what components require the MKL library ? That info might also help users know when they should install MKL before using CNTK